HEALTH_ENDPOINT.md

```diff
@@ -21,9 +21,9 @@ The endpoint returns a JSON object with the following structure:
       "status": "healthy|unhealthy",
       "message": "Database connection successful"
     },
-    "elasticsearch": {
+    "search": {
       "status": "healthy|unhealthy",
-      "message": "Elasticsearch connection successful"
+      "message": "Search connection successful"
```

Comment:
Can "Typesense" be mentioned?

Reply:
We can mention it, but it was to be consistent with the "database" key (2 lines above) that does not mention PostgreSQL either